### PR TITLE
Feat: Navigation & Rating

### PR DESCRIPTION
**Pull request by purohitdheeraj**
This branch contains the following Components
- Navigation(desktop)
- Rating input bar

Other than that did some changes to the documentation sidebar
- given margin-block-end to aside.

margin-block

### CODE REVIEW

File: docs/docs.css [hunk 1]
```diff
@@ -17,10 +17,11 @@ aside{
 }
 .container__left nav{
```
Comment (sanjitsarkar) [inline docs/docs.css:18]: use proper naming convention for naming css classes as you have used both naming method like using double underscore and dash

**Review by sanjitsarkar — APPROVED** [review]

File: docs/docs.css (hunk 1 above)
Comment (humzahasan) [reply docs/docs.css:18]: +1 on this, chose either one and stay consistant in whole codebase

File: library/components/rating/rating.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+   &lt;label for=&quot;2-stars&quot; class=&quot;star&quot;&gt;&amp;#9733;&lt;/label&gt;
+   &lt;input type=&quot;radio&quot; id=&quot;1-star&quot; name=&quot;rating&quot; value=&quot;1&quot;&gt;
+   &lt;label for=&quot;1-star&quot; class=&quot;star&quot;&gt;&amp;#9733;&lt;/label&gt;
+&lt;/div&gt;
+</code>
+
+</pre>
+        </div>
+
+
```
Comment (humzahasan) [inline library/components/rating/rating.html]: Remove extra spaces.

**Review by humzahasan — APPROVED** [review]
Code looks good to merge!

File: docs/docs.css (hunk 1 above)
Comment (purohitdheeraj) [reply docs/docs.css:18]: Sure will follow one convention and replace this thing
thanks for pointing that out 👍

**Pull request merged**